QUARKUS-1407.md

```diff
@@ -0,0 +1,38 @@
+# Quarkus-1407 - Testing and verifying Oracle SQL client
+
+The goal of this test plan is to cover the main points Oracle SQL client in Quarkus. 
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-1407 
+
+## Related TPs
+- [QUARKUS-190](QUARKUS-190.md)
+- [QUARKUS-1079](QUARKUS-1079.md)
+- [QUARKUS-1164](QUARKUS-1164.md)
+
+## Scope of the testing
+Following actions were taken to ensure test coverage and automation for vertx/sql applications:
+ - Setup databases and load initial data through Flyway
+ - Programmatic insert statements
+ - Select statements
+ - Transactional statements between several tables
+ - Tested databases: Oracle
+
+## Impact on testsuites and testing automation:
+ - Ensure this coverage works both in JVM and NATIVE mode (NB: some existing tests for OracleDb are disabled in Native mode)
```

Comment:
They're disabled in native because of FIPS, right? This should probably not be an issue down the road, and if it is, it should be tracked in Jira.

Reply:
I rechecked, and it looks like they are already enabled.